app/src/app/search/components/search.tsx

```diff
@@ -5,20 +5,23 @@ import SearchResultTable from "@/app/search/components/search-result-table";
 import {ExportCSVLink} from "@/app/search/components/search-export-csv-link";
 import {SearchProvider} from "@/app/search/search-provider";
 import {SearchResultCount} from "@/app/search/components/search-result-count";
+import SearchResultPagination from "./search-result-pagination";
 
 interface SearchProps {
   readonly regions: Tables<"region_used">[]
   readonly activityCategories: Tables<"activity_category_available">[]
   readonly statisticalVariables: Tables<"stat_definition">[]
   readonly searchFilters: SearchFilter[]
   readonly searchOrder: SearchOrder
+  readonly searchPage: SearchPage
```

Comment:
The property name `searchPage` is ambiguous and could be confused with the `SearchPage` server component. A better name could be `searchPagination` or simply `pagination`.